DataFormats/L1TCorrelator/src/classes_def.xml

```diff
@@ -9,8 +9,8 @@
   <class name="edm::Ref<std::vector<l1t::TkPrimaryVertex>,l1t::TkPrimaryVertex,edm::refhelper::FindUsingAdvance<std::vector<l1t::TkPrimaryVertex>,l1t::TkPrimaryVertex> >"/>
   <class name="std::vector<edm::Ref<std::vector<l1t::TkPrimaryVertex>,l1t::TkPrimaryVertex,edm::refhelper::FindUsingAdvance<std::vector<l1t::TkPrimaryVertex>,l1t::TkPrimaryVertex> > >"/>
 
-  <class name="l1t::TkEtMiss" ClassVersion="3">
-   <version ClassVersion="3" checksum="77762919"/>
+  <class name="l1t::TkEtMiss" ClassVersion="5">
```

Comment:
Why delete the `ClassVersion="3"`? Also, what happened to `ClassVersion="4"`?

Reply:
Error on my part, this has been resolved